wormhole-connect/src/views/Bridge/Inputs.tsx/From.tsx

```diff
@@ -82,17 +82,17 @@ function FromInputs() {
       error={!!(showErrors && validations.token)}
       onClick={openTokensModal}
       editable={!isTransactionInProgress}
-      disabled={!fromNetwork || !wallet.address}
+      disabled={!fromNetwork || !wallet.address || isTransactionInProgress}
     />
   );
 
   // amount input jsx
   const amountInput = (
     <Input
       label="Amount"
-      editable={!isTransactionInProgress}
       error={!!(showErrors && validations.amount)}
       onClick={focusAmt}
+      editable={!isTransactionInProgress}
```

Comment:
Same here, leave editable and set `disabled={!isTransactionInProgress`

Reply:
It'd show like this:

![image](https://user-images.githubusercontent.com/11276821/235223297-48e9b42e-d85c-4b1e-aef0-fb0463818c3e.png)

I changed it initially, but then I thought it would cause more confusion than anything. If you think it's fine, then I'll keep it.

Reply:
I want to keep the color fill in both asset and amount on the "From" section